moped-editor/src/views/projects/projectView/ProjectComponents/utils.js

```diff
@@ -0,0 +1,66 @@
+import { useMemo } from "react";
+import booleanIntersects from "@turf/boolean-intersects";
+import circle from "@turf/circle";
+
+/* Filters a feature collection down to one type of geometry */
+export const useFeatureTypes = (featureCollection, geomType) =>
+  useMemo(() => {
+    const features = featureCollection.features.filter((feature) => {
+      const thisGeom = feature.geometry.type.toLowerCase();
+      return thisGeom.includes(geomType);
+    });
+    return { type: "FeatureCollection", features };
+  }, [featureCollection, geomType]);
+
+/*
+Bit of a hack to generate intersection labels from nearby streets. 
+this relies on nearby lines being avaialble in-memory, which 
```

Comment:
available

Reply:
🔍 Thanks Frank! I just fixed this in another PR after going through your feedback here. https://github.com/cityofaustin/atd-moped/commit/f0fe84003039b582f30236c6fede0064874013fe